Sources/SwifterSwift/Foundation/NSAttributedStringExtensions.swift

```diff
@@ -14,10 +14,29 @@ import AppKit
 // MARK: - Properties
 
 public extension NSAttributedString {
-    #if os(iOS)
-    /// SwifterSwift: Bolded string.
+    #if canImport(UIKit)
+    private typealias Font = UIFont
+    #endif
+
+    #if canImport(AppKit) && !targetEnvironment(macCatalyst)
+    private typealias Font = NSFont
+    #endif
```

Comment:
As a convention we might want to put these at the top of the file and make them `public`. I believe the compiler complains if you try to add a `Font` parameter to a func with a private typealias. I ran into this while experimenting with `applying`.

Reply:
I'd rather avoid doing that as it would conflict with [SwiftGen](https://github.com/SwiftGen/SwiftGen). But then again our `Color` definition also clashes with it, so... 🤷‍♂️ The SwiftGen configuration allows for changing the generated enum names, so it should be fine.

Reply:
Hm. Good point. I wonder if the typealiases are namedspaced to the module. I have no clue, honestly, but I don't recall any conflicts when we pulled SwifterSwift into my app, which uses SwiftGen. If we think it's an issue maybe we should rename those to SColor/SFont? Just an idea.